docs/FAQ.md

```diff
@@ -1,48 +1,109 @@
-## COMMON QUESTIONS
+# Frequently Asked Questions
 
-#### 1 - How can I contribute? I am not a developer, I know nothing useful.
+If you have trouble installing or compiling AzerothCore read [Common Errors](common-errors.md).
 
-- You can help us by testing our [Pull Requests](Contribute#how-to-test-a-pull-request), and participating on the github issues' discussions. You can also provide support when you can in our discord, on the popular [stackoverflow](https://stackoverflow.com/questions/tagged/azerothcore) and on other emulation forums (ac-web.org for example). You can also help us with documentation (this wiki, or the readme), we need help with that too!
-- You can put bounties (money) on issues and suggestions by using bountysource. Read this: [Bountysource](Bountysource.md)
-- If you're a developer, check out this page: [Contribute](Contribute.md)
+| Did this FAQ not answer your questions? Read [How to ask for help](how-to-ask-for-help.md) on how to proceed with your question in the best way. |
+| --- |
 
-#### 2 - Why is AzerothCore using more RAM than TrinityCore?
```

Comment:
was this moved somewhere or just removed? I'd keep it tbh

Reply:
#13